### ISSUE TITLE
Error while updating password policy from Console

### ISSUE THREAD

**Issue by rksk** [wso2/product-is#18003]
**Describe the issue:**
Goto  Login & registration > Password Validation on Console and try to update password policy. The API returns following though the new policy is persisted with the next API call to `/t/carbon.super/api/server/v1/validation-rules`

```
PATCH /t/carbon.super/api/server/v1/identity-governance/UGFzc3dvcmQgUG9saWNpZXM/connectors

{
    "code": "IDG-50009",
    "message": "Resource not found.",
    "description": "Unable to find any connector with the provided identifier cGFzc3dvcmRFeHBpcnlWMg.",
    "traceId": "2937c8c1-3fbc-498d-8b62-7f5d8997ea0f"
}
```

**Environment information** 
 - Product Version: IS 7.0 beta

**Comment by ashanthamara** [comment]
Password Expiry connector name has renamed recently with this PR (https://github.com/wso2-extensions/identity-governance/pull/779/files). Hence we need to update the connector id according to that. It will resolve this issue.

**Comment by NipuniBhagya** [comment]
Verified in rc1-snapshot released on 06.12.2023
